### ISSUE TITLE
Not recommended, only for proving R*s code is shit

### ISSUE THREAD

**Issue by NjuuOwO** [tostercx/GTAO_Booster_PoC#26]
I would not recommend to play with it on your main account, I got banned after like 2 days of using it.
Yes, I know it is mentioned in README

**Comment by Wiydan** [comment]
You got bad luck then, I've been using it a week with no issues.

**Comment by pcamposu** [comment]
Yeah, better complain this PoC than complaining about the paid mod menu
